[2023-11-08T02:41:57.848+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: recalls_etl_v2.get_raw_data manual__2023-11-08T02:41:55.556062+00:00 [queued]>
[2023-11-08T02:41:57.860+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: recalls_etl_v2.get_raw_data manual__2023-11-08T02:41:55.556062+00:00 [queued]>
[2023-11-08T02:41:57.860+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-08T02:41:57.882+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): get_raw_data> on 2023-11-08 02:41:55.556062+00:00
[2023-11-08T02:41:57.891+0000] {standard_task_runner.py:57} INFO - Started process 608 to run task
[2023-11-08T02:41:57.895+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'recalls_etl_v2', 'get_raw_data', 'manual__2023-11-08T02:41:55.556062+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/recalls_etl_v2.py', '--cfg-path', '/tmp/tmp6lplm4_y']
[2023-11-08T02:41:57.897+0000] {standard_task_runner.py:85} INFO - Job 30: Subtask get_raw_data
[2023-11-08T02:41:57.961+0000] {task_command.py:416} INFO - Running <TaskInstance: recalls_etl_v2.get_raw_data manual__2023-11-08T02:41:55.556062+00:00 [running]> on host d2f41c2a3488
[2023-11-08T02:41:58.062+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='recalls_etl_v2' AIRFLOW_CTX_TASK_ID='get_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-08T02:41:55.556062+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-08T02:41:55.556062+00:00'
[2023-11-08T02:41:59.524+0000] {logging_mixin.py:151} INFO - <Response [200]>
[2023-11-08T02:41:59.579+0000] {logging_mixin.py:151} INFO - Added 29232 recalled records.
[2023-11-08T02:41:59.895+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-08T02:41:59.908+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=recalls_etl_v2, task_id=get_raw_data, execution_date=20231108T024155, start_date=20231108T024157, end_date=20231108T024159
[2023-11-08T02:41:59.938+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-08T02:41:59.969+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
